src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -391,7 +392,8 @@ export default class AirportModel {
             return;
         }
 
-        this._positionModel = new StaticPositionModel(gpsCoordinates, null, magneticNorth);
+        this._positionModel = new StaticPositionModel(gpsCoordinates, null);
+        Compass.magneticNorth = magneticNorth;
```

Comment:
It makes me nervous to see this so easily manipulated. Which is probably irrational, because we would never try to... idk, is there even a better way to do this? @n8rzz

Reply:
agreed.  this should be updated via a method or, at the very least, a settor that sets a private property.